### PR TITLE
[5.1] tinymce hardening fix

### PR DESCRIPTION
**Pull request by brianteeman**
This addresses two issues resolved in tinymce 7 that we can backport to 6.8

I checked with @joomla/security before posting

Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: plugins/editors/tinymce/src/PluginTraits/DisplayTrait.php [hunk 1]
```diff
@@ -459,6 +459,10 @@ public function display(string $name, string $content = '', array $attributes =
                 'branding'  => false,
                 'promotion' => false,
 
+                // Hardened security
+                'sandbox_iframes' => true,
+                'convert_unsafe_embeds ' => true,
```
Comment (Quy) [inline plugins/editors/tinymce/src/PluginTraits/DisplayTrait.php]:
```suggestion
                'convert_unsafe_embeds' => true,
```

File: plugins/editors/tinymce/src/PluginTraits/DisplayTrait.php [hunk 2]
```diff
@@ -459,6 +459,10 @@ public function display(string $name, string $content = '', array $attributes =
                 'branding'  => false,
                 'promotion' => false,
 
+                // Hardened security
+                'sandbox_iframes' => true,
+                'convert_unsafe_embeds' => true,
```
Comment (richard67) [inline plugins/editors/tinymce/src/PluginTraits/DisplayTrait.php]:
```suggestion
                'sandbox_iframes'       => true,
                'convert_unsafe_embeds' => true,
```
Fix PHPCS failing in Drone.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5092ce4027a06f4f0296b5dfe0a43bf07773706f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43176">issues.joomla.org/tracker/joomla-cms/43176</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43176">issues.joomla.org/tracker/joomla-cms/43176</a>.</sub>

**Comment by brianteeman** [comment]
I made this pr for 5.1 on the assumption that there will not be anothher 5.0 release. If that is incorrect then it should be backported to 5.0

**Comment by bembelimen** [comment]
Thx

**Comment by N6REJ** [comment]
just an fyi, this breaks video's that are embedded with iframes!
https://forum.joomla.org/viewtopic.php?f=834&t=1008580

**Pull request merged**